Objective-C/CBLBlob.mm

```diff
@@ -382,7 +382,9 @@ - (id) cbl_toCBLObject {
 - (void) fl_encodeToFLEncoder: (FLEncoder)encoder {
     // Note: If CBLDictionary can be encoded independently of CBLDocument,
     // so there could be no extra info:
+    bool hasAttachment = true;
     FLEncoderContext* encContext = (FLEncoderContext*)FLEncoder_GetExtraInfo(encoder);
+    encContext->outHasAttachment = &hasAttachment;
```

Comment:
The pointer of hasAttachment will be invalid right after this method return.

I think it should be like:
```
bool* outHasAttachment = encContext->outHasAttachment;
if (outHasAttachment) {
    *outHasAttachment = true;
}
```